packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -32,35 +38,68 @@ export default function ListViewSidebar() {
 		}
 	}
 
-	const instanceId = useInstanceId( ListViewSidebar );
-	const labelId = `edit-post-editor__list-view-panel-label-${ instanceId }`;
+	const [ tab, setTab ] = useState( 'list-view' );
 
 	return (
 		// eslint-disable-next-line jsx-a11y/no-static-element-interactions
 		<div
-			aria-labelledby={ labelId }
+			aria-label={ __( 'List View' ) }
 			className="edit-post-editor__list-view-panel"
 			onKeyDown={ closeOnEscape }
 		>
 			<div
-				className="edit-post-editor__list-view-panel-header"
+				className="edit-post-editor__list-view-panel-header components-panel__header edit-post-sidebar__panel-tabs"
 				ref={ headerFocusReturnRef }
 			>
-				<strong id={ labelId }>{ __( 'List View' ) }</strong>
 				<Button
 					icon={ closeSmall }
 					label={ __( 'Close List View Sidebar' ) }
 					onClick={ () => setIsListViewOpened( false ) }
 				/>
+				<ul>
+					<li>
+						<Button
+							onClick={ () => {
+								setTab( 'list-view' );
+							} }
+							className={ classnames(
+								'edit-post-sidebar__panel-tab',
+								{ 'is-active': tab === 'list-view' }
+							) }
+							aria-current={ tab === 'list-view' }
+						>
+							{ __( 'List View' ) }
+						</Button>
+					</li>
+					<li>
+						<Button
+							onClick={ () => {
+								setTab( 'outline' );
+							} }
+							className={ classnames(
+								'edit-post-sidebar__panel-tab',
+								{ 'is-active': tab === 'outline' }
+							) }
+							aria-current={ tab === 'outline' }
+						>
+							{ __( 'Outline' ) }
+						</Button>
+					</li>
+				</ul>
```

Comment:
This should probably use the `TabPanel` component - https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/tab-panel/README.md

Reply:
The TabPanel component does not support a close button at the side of the tabs. That's the reason we are not using it. The same happens with the main tabs between the block and the document. I'm following the approach used there for consistency.